test/steps/kogitoapp.go

```diff
@@ -185,31 +157,145 @@ func getKogitoAppExamplesStub(namespace, contextDir string, native, persistence,
 		kogitoApp.Spec.Build.GitSource.Reference = ref
 	}
 
-	var profiles []string
-	if persistence {
-		profiles = append(profiles, "persistence")
-		kogitoApp.Spec.EnablePersistence = true
+	if err := configureKogitoAppFromTable(table, kogitoApp); err != nil {
+		return nil, err
+	}
+
+	return kogitoApp, nil
+}
+
+func configureKogitoAppFromTable(table *gherkin.DataTable, kogitoApp *v1alpha1.KogitoApp) error {
```

Comment:
can we move the parsing of datatable to kogitoapp and resources to another file ?

Reply:
We can. Not sure how we want to have the layers/responsibilities split in the framework, but I understood it that everything about parsing GoDog etc. is right now in the steps package and the framework package is "unaware" of GoDog so it receives only things like URLs, KogitoApp etc.

So I can create utils file in the steps package as well, wdyt? There will be utils needed just for steps, so it will be nicely separated.

Reply:
I am good with it.
another possibility is to rename "step" files with suffix `_steps.go` and call this one `kogitoapp_config.go` ?
@sutaakar @Sgitario wdyt ?

Reply:
Anything to split up the steps and the datatable config would help +1

Reply:
@radtriste Sounds good

Reply:
Created utils file in steps directory as well where we can put anything for transformation of data from godog to the framework.